fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -941,8 +941,8 @@ private FedoraId mintNewPid(final FedoraId fedoraId, final String slug) {
         }
         if (!isBlank(slug)) {
             pid = slug;
-        } else if (pidMinter != null) {
-            pid = pidMinter.get();
+        // else if (pidMinter != null) {
+        //    pid = pidMinter.get();
```

Comment:
This allowed users to develop their own class to mint pids, we shouldn't disable it. If it is not set, then the default pid minter is used.